### DISCUSSION TITLE
Species not detected

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#1024] [Q&A]
Hi,

I have been happily using Birdnet-Pi for a few months on a Lepotato card. I noticed that goose or a few ducks that sometimes (very rarely but it happens) fly over my home are not detected though I could hear them.

Could it be related to the fact that I am living close to Paris and that the supposed ducks or goose are not species that is common here?

I am using the 6K global model.

Thank you for any suggestions.

**Comment by Svardsten53** [comment]
Try to lower the value "Species Occurence Frequency Threshold" in the basic settings. If you click on "more info" there is a tool (Species List Tester) where you can see which value a particular species needs to be registered for the time of the year at your location. I hope this helps.

**Reply by blogdemoi** [reply]
Edit: I can't as this option is not available with the model I selected and I do not wish to use the other model as I had problems with it.

Thank you, I will try this.
